consensus/types/src/chain_spec.rs

```diff
@@ -426,6 +426,13 @@ impl ChainSpec {
         })
     }
 
+    /// Returns true if PeerDAS epoch is scheduled for some epoch less than `FAR_FUTURE_EPOCH`.
+    pub fn peerdas_scheduled(&self) -> bool {
+        self.eip7594_fork_epoch.map_or(false, |eip7594_fork_epoch| {
+            eip7594_fork_epoch <= Epoch::max_value()
```

Comment:
should be just `<` here, but if you update your branch, there's now a `is_peer_das_scheduled` function on `unstable`.